pkg/blobstore/local/closable_block_list.go

```diff
@@ -0,0 +1,80 @@
+package local
+
+import (
+	"errors"
+	"sync"
+
+	"github.com/buildbarn/bb-storage/pkg/blobstore/buffer"
+)
+
+// ErrClosedForWriting is an error code to indicate that the BlockList does not
+// accept any more write operations and ongoing write operations will fail.
+var ErrClosedForWriting = errors.New("Location record invalid")
```

Comment:
Please make sure to return a `status.Error()`. here. `errors.New()` doesn't yield errors with a proper gRPC status code. I would also recommend making it private, so that it doesn't pollute the godocs.